src/main/java/biz/netcentric/filevault/validator/aem/cloud/AemCloudValidator.java

```diff
@@ -66,36 +75,44 @@ public AemCloudValidator(boolean allowVarNodesOutsideContainers, boolean allowLi
         this.defaultSeverity = defaultSeverity;
         this.foundViolation = false;
         this.hasMutableNodes = false;
+        this.hasImmutableNodes = false;
         this.hasInstallHooks = false;
     }
 
     @Override
     public Collection<ValidationMessage> validate(@NotNull String path) {
-        if (!foundViolation && path.startsWith("/var/")) {
+        Collection<ValidationMessage> messages = new ArrayList<>();
+        if (numVarNodeViolations < MAX_NUM_VIOLATIONS_PER_TYPE && path.startsWith("/var/")) {
             // check if package itself is only used on author
             if (!allowVarNodesOutsideContainers || !isContainedInAuthorOnlyPackage(containerValidationContext)) {
                 // only emit once per package
                 foundViolation = true;
```

Comment:
`foundViolation` is not used anymore, it can be deleted.

Reply:
Indeed, thanks for the hint.

Reply:
Fixed in https://github.com/Netcentric/aem-cloud-validator/pull/13/commits/060486e7100663013db2d79aabeeb37cbd4d8def.